quickstart/202-machine-learning-moderately-secure-existing-VNet/network.tf

```diff
@@ -0,0 +1,214 @@
+/*
```

Comment:
What is the thinking with keeping this block as a comment? Should we add a comment on when to uncomment, or only reuse certain pieces?